src/components/core/constants.js

```diff
@@ -29,4 +33,42 @@ const I18N = {
     }
 }
 
-export {ROUTES, I18N} ;
+const DialogItem = props => {
```

Comment:
The DialogItem, Message, dialogsData and messagesElemets functions are not constants - better keep them in the respective components.
You can move them back to the components - just keep them separate in the same file next to the components code.

Reply:
done